src/app/components/listing/confirm/confirm-developer.jsx

```diff
@@ -351,15 +359,17 @@ function ChplConfirmDeveloper(props) {
                       </MenuItem>
                     ))}
                   </ChplTextField>
+                  <br />
                   { !isShowingComparison
                     && (
-                      <CardActions>
```

Comment:
I see the `CardActions` element is removed. Is it still being imported? Is there a reason not to put the button in a CardActions section?

Reply:
for card action element our theme puts a grey background on the card action container, i took it out so I wouldn't need to add another class to take out the grey background

Reply:
Okay. Did you remove the import?

Reply:
Yes its below

Reply:
You removed the use of it, down here around line 366. But the CardActions element is imported up at line 5 or so. If we're not using the CardActions any more, down here in the display, then we shouldn't be importing the component up at the top